pkg/apis/core/validation/shoot.go

```diff
@@ -1034,8 +1034,13 @@ func ValidateClusterAutoscaler(autoScaler core.ClusterAutoscaler, k8sVersion str
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("maxGracefulTerminationSeconds"), *maxGracefulTerminationSeconds, "can not be negative"))
 	}
 
-	if expander := autoScaler.Expander; expander != nil && !availableClusterAutoscalerExpanderModes.Has(string(*expander)) {
-		allErrs = append(allErrs, field.NotSupported(fldPath.Child("expander"), *expander, sets.List(availableClusterAutoscalerExpanderModes)))
+	if expander := autoScaler.Expander; expander != nil {
+		expanderArray := strings.Split(string(*expander), ",")
+		for _, b := range expanderArray {
+			if !availableClusterAutoscalerExpanderModes.Has(strings.TrimSpace(b)) {
```

Comment:
Why do we need `strings.TrimSpace` here? Doesn't this mean we allow passing `expanders=foo, bar`? Probably we don't want this, do we?

Reply:
I thought it might be a bit harsh, so added the `strings.TrimSpace`
but okay, will make it a bit more straightforward if we don't allow that

Reply:
Updated